plugins/woocommerce-blocks/tests/e2e/tests/product-gallery/inner-blocks/product-gallery-large-image-next-previous/product-gallery-large-image-next-previous.block_theme.spec.ts

```diff
@@ -17,22 +17,13 @@ const blockData = {
 		frontend: {},
 		editor: {
 			leftArrow: {
```

Comment:
`leftArrow` and `rightArrow` can now became a string directly without unnecessary `insideTheImage` property in between

Reply:
https://github.com/woocommerce/woocommerce/pull/54561/commits/391f2f9e72a739cf71c6f9d94333f42f8e858c21